chainkit_client.go

```diff
@@ -77,6 +82,33 @@ func (s *chainKitClient) GetBlock(ctxParent context.Context,
 	return msgBlock, nil
 }
 
+// GetBlockHeader returns a block header given the corresponding block hash.
+func (s *chainKitClient) GetBlockHeader(ctxParent context.Context,
```

Comment:
I believe you also need to add this to `testdata/permissions.json`.

Reply:
added all the other `ChainKit` methods.